### PR TITLE
Add isServiceWorkerRegistered() method

### PR DESCRIPTION
**Pull request by olegrise**
## Description

- Add a method to check is Viewer's Service Worker is registered
- The `isServiceWorkerRegistered` returns a promise. If promise is fulfilled then Service Worker is registered.
- The `_reset` method is added to support unit testing.

## Motivation and Context
Caching epic.

## How Has This Been Tested?
Tested on local machine:

- Build widget-common
- Copied /widget-common/dist folder to the /widget-image/src/components/widget-common/dist
- Build widget-image
- Use Charle proxy to load local copy of the widget-image
- Confirmed that in the Chrome console that `RiseVision.Common.Utilities.isServiceWorkerRegistered()` returns promise that is fulfilled/resolved.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "widget-common",
-  "version": "3.13.0",
+  "version": "3.13.1",
```
Comment (santiagonoguez) [inline package.json:3]: @olegrise shouldn't this change ( new functionality ) require moving to 3.14.0 ?

**Review by santiagonoguez — COMMENTED** [review]
Code looks good to me, just a minor question regarding version.

**Review by stulees — APPROVED** [review]
@olegrise 👍

File: package.json (hunk 1 above)
Comment (olegrise) [reply package.json:3]: @santiagonoguez It's not a big change. @stulees what do you think?

Comment (stulees) [reply package.json:3]: Given it's an additional _util_ function (helper) as opposed to a new module of functionality, I think it's just a minor update.

Comment (santiagonoguez) [reply package.json:3]: good !

**Review by santiagonoguez — APPROVED** [review]

**Comment by olegrise** [comment]
@stulees @santiagonoguez please review the last commit. I added `useContentSentinel` method. Unlike [implementation in the common-template](https://github.com/Rise-Vision/common-template/blob/master/src/rise-helpers.js#L295), it does **not** include the check for `isServiceWorkerRegistered()` because here `isServiceWorkerRegistered` returns promise and I didn't want to turn `useContentSentinel` into promise. Let me know if you think it's a bas idea.

**Comment by stulees** [comment]
@olegrise It's good with me 👍  I think it lends itself well to the [decision flowchart](https://docs.google.com/document/d/15hYNlR7Oyj2x8EwhI5ewbp0j4Ji1if2pEdxxFSERdco/edit#bookmark=id.d8519h9gq3w8) because a different path is taken based on just the `useContentSentinel()` check.

**Comment by olegrise** [comment]
> @olegrise It's good with me 👍 I think it lends itself well to the [decision flowchart](https://docs.google.com/document/d/15hYNlR7Oyj2x8EwhI5ewbp0j4Ji1if2pEdxxFSERdco/edit#bookmark=id.d8519h9gq3w8) because a different path is taken based on just the `useContentSentinel()` check.

Awesome. Thanks.

**Pull request merged**